test/node/bigint.test.ts

```diff
@@ -105,6 +105,39 @@ describe('BSON BigInt support', function () {
 
       it(description, test);
     }
+
+    describe('edge case tests', function () {
```

Comment:
Just a little nit. Instead of just calling these edge case tests can we better describe what we are testing here?